ui/src/app/shared/utils.ts

```diff
@@ -92,5 +92,33 @@ export const Utils = {
         } catch {
             return null;
         }
+    },
+
+    isNodeSuspended(node: models.NodeStatus): boolean {
```

Comment:
I'm not a big fan of "util" or "common" files as they tend to get large and unwieldy. Could you create a separate file for this please?